### PR TITLE
Openeo Rclient

### PR DESCRIPTION
**Pull request by Pratichhya**
Updated the content. Examples are also working, except sometimes encountering 401 Authorisation error though logged in.

### CODE REVIEW

**Comment by Pratichhya** [comment]
I believe it is ready to be reviewed, and r works very well with openeo too 😅.

File: APIs/openEO/R_Client/R.qmd [hunk 1]
```diff
@@ -57,98 +55,73 @@ connection = connect(host = "https://openeo.dataspace.copernicus.eu")
...
-After that the website will give you the instructions to go back to the R client, where your connection has logged your account in. 
-This means, that every call that comes after that via the connection variable is executed by your user account.
+Calling this method opens the system's web browser for user authentication. Upon successful authentication, a message confirming the login success will appear on the R console.
+ 
 
-## Creating a (user-defined) process
+## Creating a datacube
 
-Now that we know how to discover the back-end and how to authenticate, lets continue by creating a new batch job to process some data.
+Now, let's create fetch and process EO data using openEO. This involves working with "Datacubes," which are fundamental in openEO for representing EO data. For more information on datacubes, refer to the glossary [here](../Glossary.qmd#spatial-datacubes).
```
Comment (HansVRP) [inline APIs/openEO/R_Client/R.qmd:124]: missing coma between create and fetch

File: APIs/openEO/R_Client/R.qmd [hunk 2]
```diff
@@ -265,34 +237,25 @@ download_results(job = job, folder = "/some/folder/on/filesystem")
...
 
-In this chapter we will show a full example of an earth observation use case using the R client.
-Instead of batch job processing, we compute the image synchronously. Synchronous processing means the result is directly returned in the response, which usually works only for smaller amounts of data.
+Certainly! Here's a simplified version without pronouns:
 
+This chapter outlines a complete example of an earth observation use case using the R client.
 
-Here, we want to produce a monthly RGB composite of Sentinel 1 backscatter data over the area of Vienna, Austria for three 
-months in 2017. This can be used for classification and crop monitoring.
+The objective is to produce a monthly RGB composite of Sentinel 1 backscatter data across Vienna, Austria for three months in 2017. This composite can support tasks like classification and crop monitoring.
```
Comment (HansVRP) [inline APIs/openEO/R_Client/R.qmd:248]: I would rather use and instead of or

File: APIs/openEO/R_Client/R.qmd [hunk 3]
```diff
@@ -312,14 +275,18 @@ data = p$load_collection(id = collections$`SENTINEL1_GRD`,
                          temporal_extent = c("2017-03-01", "2017-06-01"),
                          bands = c("VV"))
 
+# apply the SAR backscatter process to the data
+datacube_sar = p$sar_backscatter(data = data, coefficient = "sigma0-ellipsoid")
+
+
 # create three monthly sub data sets, which will be merged back into a single data cube later
```
Comment (HansVRP) [inline APIs/openEO/R_Client/R.qmd:282]: later on

File: APIs/openEO/R_Client/R.qmd [hunk 4]
```diff
@@ -368,29 +335,27 @@ rescaled = p$apply(data = merge_2,
           p$linear_scale_range(x=data, inputMin = -20,inputMax = -5, outputMin = 0, outputMax = 255)
         })
 
-# export shall be format PNG
-# look at the format description
-formats$output$PNG
 
-# store the results using the format and set the create options
+# store the results using the png format and set the create options
```
Comment (HansVRP) [inline APIs/openEO/R_Client/R.qmd:339]: do you mean job options?

File: APIs/openEO/R_Client/R.qmd [hunk 5]
```diff
@@ -368,29 +335,27 @@ rescaled = p$apply(data = merge_2,
...
 
 ![](../_images/getting-started-result-example.jpg)
 
 
 ## User Defined Functions
 
-If your use case can not be accomplished with the default processes of openEO, you can define a [user defined function](../Glossary.qmd#user-defined-function-udf).
+If a usecase cannot be defined with the default processes of openEO, use can define a [user defined function](../Glossary.qmd#user-defined-function-udf).
 
 In general the processing workflow works by uploading the Python or R script into the users file directory on the back-end and reference the script via its URL or by its relational name (e.g. `/scripts/script1.R`) in the function `run_udf`. The latter function is a predefined openEO process that the back-end might provide, if UDFs are supported.
```
Comment (HansVRP) [inline APIs/openEO/R_Client/R.qmd:360]: very long sentence

**Review by HansVRP — CHANGES_REQUESTED** [review]
Very good, only minor comments. But I will also try to read it on a rendered page

File: APIs/openEO/R_Client/R.qmd (hunk 3 above)
Comment (Pratichhya) [reply APIs/openEO/R_Client/R.qmd]: rephrased

**Comment by Pratichhya** [comment]
@jdries @HansVRP a kind reminder

**Comment by jdries** [comment]
@Pratichhya  I won't have time for this, so it's fine to rely on comments by @HansVRP and then merge.

File: APIs/openEO/R_Client/R.qmd (hunk 2 above)
Comment (HansVRP) [reply APIs/openEO/R_Client/R.qmd]: yes indeed I would replace 'and' by 'or'

**Pull request merged**